core/src/com/unciv/ui/screens/worldscreen/status/StatusButtons.kt

```diff
@@ -1,44 +1,47 @@
 package com.unciv.ui.screens.worldscreen.status
 
-import com.badlogic.gdx.scenes.scene2d.ui.HorizontalGroup
+import com.badlogic.gdx.scenes.scene2d.ui.Table
 import com.badlogic.gdx.utils.Disposable
 
 class StatusButtons(
-    nextTurnButton: NextTurnButton,
-    autoPlayStatusButton: AutoPlayStatusButton? = null,
-    multiplayerStatusButton: MultiplayerStatusButton? = null
-) : HorizontalGroup(), Disposable {
-    var autoPlayStatusButton: AutoPlayStatusButton? = autoPlayStatusButton
-        set(button) {
-            autoPlayStatusButton?.remove()
-            field = button
-            if (button != null) {
-                addActorAt(0, button)
-            }
-        }
-    var multiplayerStatusButton: MultiplayerStatusButton? = multiplayerStatusButton
-        set(button) {
-            multiplayerStatusButton?.remove()
-            field = button
-            if (button != null) {
-                addActorAt(0, button)
-            }
-        }
+    val nextTurnButton: NextTurnButton
+) : Table(), Disposable {
+    var autoPlayStatusButton: AutoPlayStatusButton? = null
+    var multiplayerStatusButton: MultiplayerStatusButton? = null
+    var smallUnitButton: SmallUnitButton? = null
+    private val padXSpace = 10f
+    private val padYSpace = 5f
     
-
     init {
-        space(10f)
-        right()
-        wrapReverse()
-        wrapSpace(10f)
-        rowRight()
-        if (autoPlayStatusButton != null) {
-            addActor(autoPlayStatusButton)
-        }
-        if (multiplayerStatusButton != null) {
-            addActor(multiplayerStatusButton)
+        add(nextTurnButton)
+    }
+    
+    fun update(portrait: Boolean) {
```

Comment:
Parameter `portrait` is misleading, better use `wrap` or `wrapped` or `vertical`.

Reply:
Fixed